integrations/gcp/src/commonMain/kotlin/com/xebia/functional/xef/gcp/GCPChatScopeExtensions.kt

```diff
@@ -0,0 +1,25 @@
+package com.xebia.functional.xef.gcp
+
+import com.xebia.functional.xef.conversation.AiDsl
+import com.xebia.functional.xef.conversation.Conversation
+import com.xebia.functional.xef.llm.Chat
+import com.xebia.functional.xef.prompt.Prompt
+import kotlinx.coroutines.flow.Flow
+
+@AiDsl
+suspend fun Conversation.promptMessage(
+  prompt: Prompt,
+  model: Chat,
+): String = model.promptMessage(prompt)
+
+@AiDsl
+suspend fun Conversation.promptMessage(
```

Comment:
I'm not sure if we need this function for now. I think that we only need the previous one

Reply:
I included it because this corresponds to the equivalent function in the OpenAI file where the function has a default argument (which we cannot provide since this depends on GCP). I thought better have it than not.